app/scripts/app.js

```diff
@@ -80,6 +80,11 @@ angular
             'pageTitle': event.targetScope.ngMeta.title
           });
         }
+        window.parent.postMessage({
+          source: 'UDB',
+          type: 'URL_CHANGE',
```

Comment:
Nit: Maybe we should consider using the past tense so it’s like an event dispatcher/listener? But it’s only one message type so far so we can always revisit this once we add others and have a better view of what types we’ll have. Just wanted to already share the idea.